src/Components/Users/UserProfile.tsx

```diff
@@ -22,32 +33,251 @@ const useStyles = makeStyles((theme: Theme) =>
         },
     }),
 );
+const initForm: any = {
+    first_name: "",
```

Comment:
Codacy found an issue: [Identifier 'first_name' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5815458)